app/assets/javascripts/events.coffee

```diff
@@ -13,8 +14,10 @@ $ () ->
       itemSelector: '.event'
     },
     (newEvents) ->
+      $spinner.show()
```

Comment:
it means spinner will be shown when we get new events from the server, so if there will be some delay, user will not see the spinner during request flow.